### PR TITLE
meta: Update Github PR/Issue templates

### PR DESCRIPTION
**Pull request by hydrobeam**
# Related Issues

N/A

# Summary of Changes

Refactor github templates.

### `feature request` + `bug_report`
- used comments instead of italics text to indicate the behaviour the submitter is expected to follow
- used `##` instead of bold text to indicate subsections
- more concise
### `pull_request_template`
- minor re-ordering to favour `Summary of changes` over linked Issues
- added motivation and  additional comments sections

# Steps to Reproduce

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
-# Related Issues
-<!-- Issue(s) that this PR will resolve, e.g. "resolves <issue link here>". -->
-
-# Summary of Changes
+## Summary of Changes
 <!-- A summary or description of changes made in this PR. -->
 
-# Steps to Reproduce
+## Motivation and Explanation
+<!-- How and why do your changes improve the bot? -->
```
Comment (mcpenguin) [inline .github/pull_request_template.md:5]: Add a section for screenshots/video showcasing the new feature.

**Comment by mcpenguin** [comment]
Also, in the future, request a review from one of the devs when you make a PR.

**Review by mcpenguin — COMMENTED** [review]
Left some comments - otherwise LGTM

File: .github/pull_request_template.md (hunk 1 above)
Comment (hydrobeam) [reply .github/pull_request_template.md:5]: That's covered in the `Steps to Reproduce` section below isn't it?

**Comment by hydrobeam** [comment]
> Also, in the future, request a review from one of the devs when you make a PR, and re-request when you make any changes.

Which dev do I request a review from? Do I just pick at random?

**Comment by mcpenguin** [comment]
> > Also, in the future, request a review from one of the devs when you make a PR, and re-request when you make any changes.
> 
> Which dev do I request a review from? Do I just pick at random?

You can request from either me or Picowchew, since out of all the devs we check the repository the most frequently.

File: .github/pull_request_template.md (hunk 1 above)
Comment (mcpenguin) [reply .github/pull_request_template.md:5]: Make it an explicit heading - people can remove it or say "n/a" if there are no screenshots to show.

**Review by mcpenguin — CHANGES_REQUESTED** [review]
small changes, otherwise LGTM

**Review by mcpenguin — APPROVED** [review]
LGTM!

**Pull request merged**